docs/sdk-and-tools/sdk-py/mxpy-cli.md

```diff
@@ -530,35 +530,58 @@ If your address is guarded by another wallet, you'll still need to provide the `
+
+### Creating the inner transaction
+
+The inner transaction is any regular transaction, with the following notes:
+- relayer address must be added
+- extra 50000(base cost) gas must be added for the relayed operation. For more details on how the gas is computed, check out [this page](/developers/relayed-transactions/#relayed-transactions-version-3).
+
+This can be generated through `mxpy tx new` command. A new argument `--relayer` has been added for this feature.
 
 ```sh
 mxpy tx new --pem ~/multiversx-sdk/testwallets/latest/users/alice.pem --recall-nonce \
     --receiver erd1spyavw0956vq68xj8y4tenjpq2wd5a9p2c6j8gsz7ztyrnpxrruqzu66jx \
-    --gas-limit 50000 --value 1000000000000000000 \
+    --gas-limit 100000 --value 1000000000000000000 \
     --proxy https://devnet-gateway.multiversx.com --chain D \
-    --relayer erd1k2s324ww2g0yj38qn2ch2jwctdy8mnfxep94q9arncc6xecg3xaq6mjse8
-    --inner-transactions-outfile inner_transactions.json
+    --relayer erd1k2s324ww2g0yj38qn2ch2jwctdy8mnfxep94q9arncc6xecg3xaq6mjse8 \
+    --outfile inner_tx.json
 ```
 
-After creating the inner transaction, we are ready to create the relayed transaction. We have to keep in mind that for `Relayed V3` transactions, the receiver has to be the same as the relayer, in our case, Carol. Another requirement is that the relayed transaction has to have enough gas. The gas is computed by multiplying the base cost (50_000) with the number of inner transactions plus the gasLimit for each inner transaction. For more details on how the gas is computed, check out [this page](/developers/relayed-transactions/#relayed-transactions-version-3).
+After creating the inner transaction, we are ready to create the relayed transaction. We have to keep in mind that for `Relayed V3` transactions.
+
+### Creating the relayed transaction
 
 We can create the relayed transaction by running the following command:
 
 ```sh
-mxpy tx new --pem ~/multiversx-sdk/testwallets/latest/users/carol.pem --recall-nonce \
-    --receiver erd1k2s324ww2g0yj38qn2ch2jwctdy8mnfxep94q9arncc6xecg3xaq6mjse8 \
-    --gas-limit 1000000 --value 0 \
+mxpy tx relay --relayer-pem ~/multiversx-sdk/testwallets/latest/users/carol.pem \
+    --proxy https://devnet-gateway.multiversx.com --chain D \
+    --infile inner_tx.json \
+    --send
+```
+
+### Creating the relayed transaction in one step
+
+This can be done through `mxpy tx new` command, as follows:
+```sh
+mxpy tx new --pem ~/multiversx-sdk/testwallets/latest/users/alice.pem --recall-nonce \
+    --receiver erd1spyavw0956vq68xj8y4tenjpq2wd5a9p2c6j8gsz7ztyrnpxrruqzu66jx \
+    --relayer-pem ~/multiversx-sdk/testwallets/latest/users/carol.pem \
+    --gas-limit 100000 --value 1000000000000000000 \
     --proxy https://devnet-gateway.multiversx.com --chain D \
-    --inner-transactions inner_transactions.json \
+    --relayer erd1k2s324ww2g0yj38qn2ch2jwctdy8mnfxep94q9arncc6xecg3xaq6mjse8 \
```

Comment:
maybe `--relayer` and `--relayer-pem` should be next to each other for readability purpose. just my opinion.